python/packages/sdk-schema/pyproject.toml

```diff
@@ -28,6 +29,11 @@ tests = [
     "pytest>=7.2",
     "ruff>=0.0.199",
 ]
+[project.urls]
+changelog = "https://github.com/serverless/console/blob/main/python/packages/sdk-schema/CHANGELOG.md"
+documentation = "https://github.com/serverless/console#readme"
```

Comment:
Maybe we can link directly to package folder (?)

Reply:
👍   I've updated `documentation` to point to the respective package folders.